tests/reporter/kubernetes.go

```diff
@@ -95,16 +100,16 @@ func (r *KubernetesReporter) BeforeSuiteDidRun(_ *types.SetupSummary) {
 }
 
 func (r *KubernetesReporter) SpecWillRun(_ *types.SpecSummary) {
-	fmt.Fprintf(ginkgo.GinkgoWriter, "On failure, artifacts will be collected in %s/%d_*\n", r.artifactsDir, r.failureCount+1)
```

Comment:
We have to call this code to the `ReportBeforeEach`

Reply:
will print as part of the function called by
https://github.com/kubevirt/kubevirt/pull/7326/commits/2015e0e7073beb63f9599279b0e8751a3b5e4d13#diff-0a9a24f44dfc4029d6c2465ca6246bf66829e97d236da6ff80172a8a9c73866aR115

so it will print only on error, not on every test
ok ?